src/PrestaShopBundle/Resources/views/Admin/Configure/ShopParameters/Blocks/product_preferences_pagination.html.twig

```diff
@@ -55,7 +55,13 @@
             {{ form_widget(paginationForm.default_order_way) }}
           </div>
         </div>
-
+        <div class="form-group row">
+          {{ ps.label_with_help(('Display out-of-stock products last'|trans), ('If enabled, products will be displayed in this order: 1. in-stock, 2. available for order but quantity is 0, 3. out-of-stock. Note that it apply only when ordering by quantity, price or dates'|trans({}, 'Admin.Shopparameters.Help'))) }}
```

Comment:
I would not use the word 'reordering' here, it can be ambiguous. What about _Note that it only applies when filtering by quantity, price or date._ instead?